src/main/java/com/simibubi/create/content/logistics/chute/ChuteBlockEntity.java

```diff
@@ -389,10 +389,11 @@ private boolean handleDownwardOutput(boolean simulate) {
 				}
 				if (inserted != 0)
 					return true;
-				invVersionTracker.awaitNewVersion(inv);
 				if (direction == Direction.DOWN)
 					return false;
 			}
+
+			invVersionTracker.awaitNewVersion(inv);
```

Comment:
this is an incorrect change in behavior

Reply:
awaitNewVersion can't be called during a transaction so I'm not sure where it should really be called from

Reply:
move the early return too. leave a comment saying why it's outside it